frame/acronym.tex

```diff
@@ -1,12 +1,12 @@
 % Utility command to avoid duplicating the label
 \newcommand{\acr}[2]{
     \lowercase{\def\key{#1}} % Remove the lowercase part if you want to address your glossary label to have the same capitalization as your name (first parameter)
-    \newacronym{\key}{#1}{#2}
+    \newacronym{#1}{#1}{#2}
```

Comment:
geht exklusiv das lowercase key nicht bei newacronym? was genau ist da das Problem mit?

Reply:
Das problem damit scheint zu sein, dass das \key, was in das newacronym gereicht wird erst evaluiert wird, wenn das verzeichnis generiert wird. Und weil mit \def \key immer wieder überschrieben wird, steht für alle acronyme das letzte acronym im Verzeichnis. Für die controlsequence erzeugung ist das aber kein problem, weil dort \key direkt ausgewertet wird.

Reply:
okay cool, danke für den Fix